rtc/jwt-authentication.md

```diff
@@ -189,7 +189,7 @@ tinymce.init({
       body: JSON.stringify({ documentId }),
     })
     .then((response) => response.json())
-    .then(({ jwt })) => ({ token: jwt })
+    .then(({ jwt }) => ({ token: jwt }))
```

Comment:
oh that reminds me I was going to try to clarify this line for a customer. They were returning `{ token }` from their API, didn't realise this was being clever and destructuring expected a `{ jwt }` object.

I'll do it in 6 later.

Reply:
Logged as DOC-1641 and I'm about to submit a PR 🤷